lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +214,31 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        url = ''
+        urls = article_bs.findAll('a', class_='entry-header')
+        for el in urls:
+            url = el.get('href')
+            if url.split('/')[2] == 'servernews.ru':
+                continue
+            if url not in self.urls:
+                break
+        url = self.url_pattern + url
+        return url
 
     def find_articles(self) -> None:
         """
         Find articles.
         """
+        seeds = self.get_search_urls()
+        for seed in seeds:
+            response = make_request(seed, self.config)
+            if not response.ok:
+                continue
+            article_bs = BeautifulSoup(response.text, 'html.parser')
+            urls = []
+            for i in range(25):
```

Comment:
25 is too strict - seems like hardcode

why exactly this number of iterations?

Reply:
My site has 25 articles for each page, that's why I use 25 iterations

Reply:
Already fixed  the whole iteration